### ISSUE TITLE
[SF][0.21.2][ActionSheet][a11y] Cannot fire event bound to item of ActionSheet through keyboard when using JAWS

### ISSUE THREAD

**Issue by i323808** [SAP/ui5-webcomponents-react#2646]
**Describe the bug**
Cannot fire event bound to item of ActionSheet through keyboard. 

**Isolated Example**
https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-actionsheet--default-story

**To Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents-react/?path=/docs/modals-popovers-actionsheet--default-story
2. Focus on 'Open ActionSheet' and press Enter key to open action sheet
3. Press Enter key when the first item of action sheet is focused
4. The focus shifts back to 'Open ActionSheet' button and the popover remains open

**Expected behavior**
The popover should be closed.

**UI5 Web Components for React Information**
`@ui5/webcomponents` version: 1.1.2
`@ui5/webcomponents-react` version: 0.21.2
Operating System:
Browser: Chrome Version 98.0.4758.102
Windows 10
**JAWS 2022**

**Comment by Lukas742** [comment]
Hello @i323808 

I just tested it myself (Mac & Windows, Chrome, Firefox, Safari) and it seems to work for me. Could you tell me what browser and os you are using?

**Comment by i323808** [comment]
Hi @Lukas742 

This issue only can be reproduced when using JAWS.
**JAWS 2022**
OS: Windows 10
Browser: Chrome Version 98.0.4758.102

BR,
Jane
